wiki/Game_mode/id.md

```diff
@@ -0,0 +1,46 @@
+# Mode permainan
+
+osu! memiliki empat **mode permainan** yang dapat kamu mainkan: [osu!](#-osu!), [osu!taiko](#-osu!taiko), [osu!catch](#-osu!catch), dan [osu!mania](#-osu!mania). Keempat mode permainan tersebut memiliki mekanisme permainannya masing-masing yang terinspirasi dari berbagai permainan ritme lainnya yang ada di luar sana.
+
+Setiap mode permainan memiliki simbol/ikonnya masing-masing sebagai berikut: ![][o!s] ![][o!t] ![][o!c] ![][o!m] (yang secara berurutan melambangkan mode osu!, osu!taiko, osu!catch, dan osu!mania).
+
+Secara *default*, kamu akan disambut dengan mode permainan yang terakhir kali kamu mainkan setiap kali kamu membuka klien osu! milikmu. Untuk dapat berpindah dari satu mode permainan ke mode permainan lainnya, kamu dapat menggunakan tombol `Mode` yang berada pada bagian bawah layar Song Selection atau menekan kombinasi tombol `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, atau `Ctrl`+`4` untuk berpindah ke mode osu!, osu!taiko, osu!catch, atau osu!mania.
+
+![Tampilan antarmuka layar pemilihan mode permainan](/wiki/shared/Interface_mode.png "Tampilan antarmuka layar pemilihan mode permainan")
+
+## ![][o!s] osu!
+
+*Laman utama: [osu!](osu!)*
+
+*osu!* (terkadang juga disebut dengan *osu!standard*) merupakan mode permainan utama yang dimiliki oleh osu!. Mode permainan ini didasarkan pada permainan *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* yang dirilis oleh [iNiS](https://en.wikipedia.org/wiki/INiS "Wikipedia"). Dalam mode permainan ini, kamu dituntut untuk dapat mengenai semua circle, slider, dan spinner yang ada mengikuti irama lagu.
+
+![Tampilan permainan osu!](/wiki/shared/osu-gameplay.jpg "Permainan osu!")
+
+## ![][o!t] osu!taiko
+
+*Laman utama: [osu!taiko](osu!taiko)*
+
+*osu!taiko* merupakan mode permainan yang didasarkan pada seri permainan *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* yang dirilis oleh [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). Dalam mode permainan yang dibuat untuk menyerupai permainan drum ini, kamu dituntut untuk dapat mengenai not-not yang muncul pada layar sesuai dengan indikator warnanya masing-masing. 
```

Comment:
```suggestion
*osu!taiko* merupakan mode permainan yang didasarkan pada seri permainan *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* yang dirilis oleh [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). Dalam mode permainan yang dibuat untuk menyerupai permainan drum ini, kamu dituntut untuk dapat mengenai note yang muncul pada layar sesuai dengan indikator warnanya masing-masing. 
```

Reply:
common using to `note` instead.

Reply:
`not` (instead of `note`) is actually the correct term here tho i believe 

![image](https://user-images.githubusercontent.com/36564236/113478463-23772600-94b3-11eb-80dc-90edb5c25503.png)